QueryBuilder/Compilers/Compiler.cs

```diff
@@ -16,14 +16,34 @@ protected Compiler(ISqlResultBinder sqlResultBinder)
             _sqlResultBinder = sqlResultBinder;
             _compileConditionMethodsProvider = new ConditionsCompilerProvider(this);
         }
-        
+
         public abstract string EngineCode { get; }
         protected string OpeningIdentifier = "\"";
         protected string ClosingIdentifier = "\"";
         protected string ColumnAsKeyword = "AS ";
         protected string TableAsKeyword = "AS ";
         protected string LastId = "";
 
+        /// <summary>
+        /// A list of white-listed operators
+        /// </summary>
+        /// <value></value>
+        protected readonly HashSet<string> operators = new HashSet<string>
+        {
+            "=", "<", ">", "<=", ">=", "<>", "!=", "<=>",
+            "like", "not like",
+            "ilike", "not ilike",
+            "like binary", "not like binary",
+            "rlike", "not rlike",
+            "regexp", "not regexp",
+            "similar to", "not similar to"
+        };
+
+        protected HashSet<string> userOperators = new HashSet<string>
```

Comment:
What is the purpose of this variable? You can already alter `operators` since it's contents are not readonly